provision-contest/ansible/roles/domserver/tasks/main.yml

```diff
@@ -112,3 +112,15 @@
   loop:
     - nginx
     - php{{ php_version.stdout }}-fpm
+
+- name: Send errors to Sentry
+  lineinfile:
+    regexp: '^SENTRY_DSN='
+    state: present
+    line: "SENTRY_DSN={{ SENTRY_DSN | default('') }}"
+    dest: "{{ DJ_DIR }}/webapp/.env.local"
+    create: true
+    mode: 0664
+    group: domjudge
+    owner: domjudge
+  notify: Restart PHP FPM
```

Comment:
Do we need a restart of FPM?

Reply:
I tested again and it seems we don't need it.